src/gql/GQLClientProvider.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
```

Comment:
It's not clear to me what the difference is between this wrapper and the GQLWrapper. Aren't they both an apollo client wrapper around the app? Could they be combined into 1 component?

Reply:
@john-m-liu I wanted to keep the environment variable accessor function separate from GQLWrapper for when/if we decide to move GQLWrapper into evergreen.js

Reply:
Having separation makes sense, but does this wrapper actually do anything? Or to put it differently, if https://github.com/evergreen-ci/spruce/pull/34/files/67875b0400f887763004d1275e945851f8be91f0#diff-6403a7f3b1429f6cdbaef0a7240047bcR67 is a GQLWrapper component, does that change anything?

Reply:
we can do away with GQLClientProvider